README.md

```diff
@@ -80,18 +80,24 @@ compilation itself expensive, but this also prevents optimizations that reuse
 allocations internally to the matching engines.
 
 In Rust, it can sometimes be a pain to pass regular expressions around if
-they're used from inside a helper function. Instead, we recommend using the
-[`once_cell`](https://crates.io/crates/once_cell) crate to ensure that
-regular expressions are compiled exactly once. For example:
+they're used from inside a helper function. Instead, we recommend using
+[`std::sync::LazyLock`] stabilized in [Rust 1.80], the [`once_cell`] crate, or
+the [`lazy_static`] crate to ensure that regular expressions are compiled
+exactly once. For example:
```

Comment:
Is there a reason not to just use the `LazyLock` option? As a user without much idea on the history, it would be nice if there was just one standard and recommended option, the stdlib one.

Also the one that clippy recommends: https://rust-lang.github.io/rust-clippy/master/index.html#non_std_lazy_statics

Reply:
@gibfahn
Thanks for your comment.
As `regex` and `once_cell` can be used with `no_std` where `std::sync::LazyLock` is not available, I thought it would be a good idea to leave mention of `once_cell`.
Since I have not tried it, if I am wrong please let me know.